ModelBundle/DataFixtures/MongoDB/LoadContentData.php

```diff
@@ -19,15 +19,11 @@ class LoadContentData extends AbstractFixture implements OrderedFixtureInterface
      */
     public function load(ObjectManager $objectManager)
     {
-        $objectManager->persist($this->generateCarR5('fr'));
-        $objectManager->persist($this->generateCarR5('en'));
-        $objectManager->persist($this->generateCar206('fr'));
-        $objectManager->persist($this->generateCar206('en'));
-        $objectManager->persist($this->generateCarDs3('fr'));
-        $objectManager->persist($this->generateCarDs3('en'));
-
-        $objectManager->persist($this->generateCustomerConvenant('fr'));
-        $objectManager->persist($this->generateCustomerConvenant('en'));
+        $languages = array('fr', "en");
+        $this->persistMultiLanguageContent($objectManager,"generateCarR5", $languages);
```

Comment:
you could set the $objectManager as an attribute of the LoadContentData, this way you can reduce the call not passing it and use it in persistMultiLanguageContent with $this->objectManager